packages/catalog-model/src/validation/KubernetesValidatorFunctions.ts

```diff
@@ -49,7 +49,7 @@ export class KubernetesValidatorFunctions {
       typeof value === 'string' &&
       value.length >= 1 &&
       value.length <= 63 &&
-      /^([A-Za-z0-9][-A-Za-z0-9_.]*)?[A-Za-z0-9]$/.test(value)
+      /^([A-Za-z0-9][-A-Za-z0-9_.]*)?[A-Za-z0-9-]$/.test(value)
```

Comment:
you can probably remove the last matcher entirely right?

`^[A-Za-z0-9][-A-Za-z0-9_.]*$`

just on a hunch, untested :)

Reply:
hmm, I'm sorry I'm not a regex expert as well, was just checking a few cases with that character and it works :D are you referring to updating this line to your suggestion?

Reply:
Yes. But it seems that the tests fail now, check their output below.